Example/PrebidDemoJava/src/main/java/org/prebid/mobile/javademo/activities/ads/gam/original/GamOriginalApiDisplayInterstitial.java

```diff
@@ -0,0 +1,70 @@
+package org.prebid.mobile.javademo.activities.ads.gam.original;
+
+import android.os.Bundle;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.android.gms.ads.LoadAdError;
+import com.google.android.gms.ads.admanager.AdManagerAdRequest;
+import com.google.android.gms.ads.admanager.AdManagerInterstitialAd;
+import com.google.android.gms.ads.admanager.AdManagerInterstitialAdLoadCallback;
+
+import org.prebid.mobile.AdUnit;
+import org.prebid.mobile.InterstitialAdUnit;
+import org.prebid.mobile.PrebidMobile;
+import org.prebid.mobile.javademo.activities.BaseAdActivity;
+
+public class GamOriginalApiDisplayInterstitial extends BaseAdActivity {
+
+    private static final String AD_UNIT_ID = "/21808260008/prebid-demo-app-original-api-display-interstitial";
+    private static final String CONFIG_ID = "imp-prebid-display-interstitial-320-480";
+    private static final String STORED_RESPONSE = "response-prebid-display-interstitial-320-480";
+
+    private AdUnit adUnit;
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        // The ID of Mocked Bid Response on PBS. Only for test cases.
+        PrebidMobile.setStoredAuctionResponse(STORED_RESPONSE);
+
+        createAd();
+    }
+
+    private void createAd() {
+        adUnit = new InterstitialAdUnit(CONFIG_ID);
+        adUnit.setAutoRefreshInterval(getRefreshTimeSeconds());
```

Comment:
not critical, but we do not need refresh intervals for interstitial ads. Please remove it in the scope of other tasks